vmm_tests/vmm_tests/tests/tests/x86_64/openhcl_linux_direct.rs

```diff
@@ -396,6 +396,90 @@ async fn openhcl_linux_storvsp_dvd(config: PetriVmConfigOpenVmm) -> Result<(), a
     Ok(())
 }
 
+/// Test an OpenHCL Linux direct VM with a SCSI DVD assigned to VTL2, using NVMe
+/// backing, and vmbus relay. This should expose a DVD to VTL0 via vmbus.
+#[openvmm_test(openhcl_linux_direct_x64)]
+async fn openhcl_linux_storvsp_dvd_nvme(config: PetriVmConfigOpenVmm) -> Result<(), anyhow::Error> {
+    const NVME_INSTANCE: Guid = guid::guid!("dce4ebad-182f-46c0-8d30-8446c1c62ab3");
+    let vtl2_nsid = 1;
+    let nvme_disk_sectors: u64 = 0x10000;
+    let sector_size = 512;
+
+    let vtl2_lun = 5;
+    let scsi_instance = Guid::new_random();
+
+    let mut vtl2_settings = None;
+
+    let (vm, agent) = config
+        .with_vmbus_redirect()
+        .with_custom_config(|c| {
+            c.vpci_devices.extend([VpciDeviceConfig {
```

Comment:
I'm tempted to ask you to figure out a way to abstract this out to make it more concise (we have multiple places now creating nvme disks with mostly the same parameters). But, if we're going to end here then that's probably over-factoring for the sake of elegance. What do you think?

Reply:
We would need to account for all the possible configurations of NVMe disks, such as striped, etc., but I think this may be good future work.

Reply:
vmm_test_macros doesn't quite seem like the right place for this helper and I don't want to put it directly in the tests file. Any thoughts?